DataFormats/FWLite/interface/Event.h

```diff
@@ -159,6 +161,8 @@ namespace fwlite {
          fwlite::LuminosityBlock const& getLuminosityBlock() const;
          fwlite::Run             const& getRun() const;
 
+         TBranch* getAuxBranch() const { return auxBranch_; }
```

Comment:
I do not like exposing this internal detail, especially return a non const pointer from a const function. What functionality requires this change?